libs/commands/src/exceptions/no-command-handler-registered-for-command-exception.ts

```
@@ -1,7 +1,7 @@
 export class NoCommandHandlerRegisteredForCommandException extends Error {
     constructor(commandType: string) {
         super(
-            `There is no handler registered for the command with type: ${commandType}. \nDid you forget the decorator on your command handler?`
+            `There is no handler registered for the command with type: ${commandType}. \nDid you decorate your command handler? \n If so, did you register your command and handler in the relevant domain module?`
```

Awesome! Gotta love failing fast.